django/db/backends/oracle/creation.py

```diff
@@ -77,9 +77,6 @@ def _create_test_db(self, verbosity=1, autoclobber=False, keepdb=False):
             try:
                 self._create_test_user(cursor, parameters, verbosity, keepdb)
             except Exception as e:
-                # If we want to keep the db, then we want to also keep the user.
-                if keepdb:
-                    return
```

Comment:
Should this be treated as part of the same ticket? I'm not sure if there's a risk of a regression if it's backported to the stable branches.